FrameworkSystem/private/standardLogging/Backend/StderrBackend.py

```diff
@@ -0,0 +1,30 @@
+"""
+StderrBackend wrapper
+"""
+
+__RCSID__ = "$Id$"
+
+import logging
+import sys
+
+from DIRAC.FrameworkSystem.private.standardLogging.Backend.Backend import Backend
+from DIRAC.FrameworkSystem.private.standardLogging.Formatter.BaseFormatter import BaseFormatter
```

Comment:
Yes, I thought it was a problem before, but with Backend objects, I can easily keep the distinction between stdout and stderr, so it is not a problem to not use color for stderr. 
Do you want to use color for stderr ?